### DISCUSSION TITLE
Which server am I using?

### DISCUSSION THREAD

**Discussion by buhtz** [joaotavora/eglot#938] [Ideas]
In the Messages and Eglot buffer I am not able to see which server eglot is connected to.

I do not understand the details of lsp protocol. But if it is possible for eglot to know (or ask) it would be nice if the server name (and version etc) would be logged into one of the buffers.

**Comment by joaotavora** [comment]
You might have some luck typing M-x eglot-events-buffer and inspecting the
communication transcript. Some servers log their id there.
